dpcpp/matrix/csr_kernels.dp.cpp

```diff
@@ -1108,9 +1108,7 @@ void classical_spmv(syn::value_list<int, subgroup_size>,
                     const matrix::Dense<ValueType>* alpha = nullptr,
                     const matrix::Dense<ValueType>* beta = nullptr)
 {
-    constexpr int threads_per_cu = 7;
-    const auto num_subgroup =
-        exec->get_num_computing_units() * threads_per_cu * classical_overweight;
+    const auto num_subgroup = exec->get_num_warps() * classical_overweight;
```

Comment:
if we're already working on this, maybe `oversubscription` is a more precise term?

Reply:
do you mean the classical_overweight?

Reply:
Agree in using `classical_oversubscription` or similar.